openslides_backend/services/media/interface.py

```diff
@@ -6,10 +8,13 @@ class MediaService(Protocol):
     Mediaservice defines the interface to the mediaservice.
     """
 
-    def upload(
+    def upload_mediafile(
```

Comment:
`upload_resource` is missing in the interface

Reply:
See some lines further down, there it is;-)